### PR TITLE
deploy to heroku

### PR DESCRIPTION
**Pull request by pwesferguson**

### CODE REVIEW

File: imports/api/places.js [hunk 1]
```diff
@@ -13,19 +9,26 @@ const googleMapsClient = createClient({
...
-  const { minprice, maxprice } = priceRange;
-  return googleMapsClient.placesNearby({
-    location,
-    radius,
-    minprice,
-    maxprice,
-    type
-  }).asPromise();
+export default function getNearbyPlaces(location, radius, budgetRange, type) {
+  const [minprice, maxprice] = budgetRange;
```
Comment (hailin-zhang) [inline imports/api/places.js]: space between [ ]'s

File: imports/api/places.js [hunk 2]
```diff
@@ -13,19 +9,26 @@ const googleMapsClient = createClient({
...
+  const { lat, lng } = location;
+  
+  const url = 'https://maps.googleapis.com/maps/api/place/nearbysearch/json?' +
```
Comment (hailin-zhang) [inline imports/api/places.js]: Airbnb styling says strings should be in the same line. But this should be okay too 😈

File: imports/redux/reducers/index.jsx [hunk 3]
```diff
@@ -185,9 +185,9 @@ function placeSearchReducer(state = initialPlaceSearchState, action) {
         typesAndQuantities: state.typesAndQuantities.set(action.placeType, action.quantity)
       };
     case REMOVE_PLACE_TYPE:
+      state.typesAndQuantities.delete(action.placeType);
```
Comment (hailin-zhang) [inline imports/redux/reducers/index.jsx]: hmm.. this seems like it's going against the immutability of state. If what you did worked before, I'd prefer that, but for now we can leave it

File: imports/ui/pages/home/components/mapContainer.jsx [hunk 4]
```diff
@@ -67,5 +68,5 @@ const mapStateToProps = (state) => {
 }
 
 export default connect(mapStateToProps)(GoogleApiWrapper({
-  apiKey: ("") // paste API key here each time, DO NOT COMMIT.
+  apiKey: (API_KEY) // paste API key here each time, DO NOT COMMIT.
```
Comment (hailin-zhang) [inline imports/ui/pages/home/components/mapContainer.jsx]: can you remove the comment!! thanks :)

File: imports/ui/pages/results/components/resultsButtons.jsx [hunk 5]
```diff
@@ -17,6 +17,7 @@ class ResultsButtons extends React.Component {
     }
 
     render() {
+        // todo remove when api call working
         const { places } = this.props;
         console.log("places:", places);
```
Comment (hailin-zhang) [inline imports/ui/pages/results/components/resultsButtons.jsx]: remove this console log please

File: imports/ui/pages/results/components/resultsMapContainer.jsx [hunk 6]
```diff
@@ -34,5 +35,5 @@ const mapStateToProps = (state) => {
 }
 
 export default connect(mapStateToProps)(GoogleApiWrapper({
-  apiKey: ("") // paste API key here each time, DO NOT COMMIT.
+  apiKey: API_KEY // paste API key here each time, DO NOT COMMIT.
```
Comment (hailin-zhang) [inline imports/ui/pages/results/components/resultsMapContainer.jsx]: remove the comment please

File: imports/ui/shared_components/navbar/navbar.css [hunk 7]
```diff
@@ -149,7 +149,8 @@
 }
 
 .slider {
-    touch-action: none; //Fixes console log errors
+  /* Fixes console log errors */
+    touch-action: none;
 }
 
 .MuiTypography-h6 {
```
Comment (hailin-zhang) [inline imports/ui/shared_components/navbar/navbar.css]: fix the indentation

Comment (hailin-zhang) [reply imports/ui/shared_components/navbar/navbar.css]: Also if there is only 1 nested ID, can't you make this a 1-line query?

Comment (pwesferguson) [reply imports/ui/shared_components/navbar/navbar.css]: i'm not sure what you mean

File: imports/api/places.js (hunk 1 above)
Comment (pwesferguson) [reply imports/api/places.js]: fixed

File: imports/redux/reducers/index.jsx (hunk 3 above)
Comment (pwesferguson) [reply imports/redux/reducers/index.jsx]: nah I think you're right, check changes

File: imports/ui/pages/home/components/mapContainer.jsx (hunk 4 above)
Comment (pwesferguson) [reply imports/ui/pages/home/components/mapContainer.jsx]: fixed

File: imports/ui/pages/results/components/resultsButtons.jsx (hunk 5 above)
Comment (pwesferguson) [reply imports/ui/pages/results/components/resultsButtons.jsx]: fixed

File: imports/ui/pages/results/components/resultsMapContainer.jsx (hunk 6 above)
Comment (pwesferguson) [reply imports/ui/pages/results/components/resultsMapContainer.jsx]: fixed

File: imports/ui/shared_components/navbar/navbar.css (hunk 7 above)
Comment (hailin-zhang) [reply imports/ui/shared_components/navbar/navbar.css]: NVM loool

**Pull request merged**